### PR TITLE
Cell temperatures

### PR DESCRIPTION
**Pull request by jatron**
Implemented thermistor voltage readings.

### CODE REVIEW

**Comment by ericponce** [comment]
I think the code could be improved for GetCellTemepratures by doing something like:

addr = getthermaddr();
for (j = 7; j >= 0; j++) {
SetGPIO(DATA, (addr & (1 << j)) >> j);
SetGPIO(CLOCK, 0);
SetGPIO(CLOCK, 1);
}

SetGPIO(LATCH, 0);
SetGPIO(LATCH, 1);

As of now it seems unnecessarily long and requires a function call to get a bit (which can be done with some logic)

**Comment by ericponce** [comment]
Also you should really check for errors from LTC drivers like we do in GetCellVoltages because it hooks into the error handler

File: src/board.c [hunk 1]
```diff
@@ -477,71 +479,182 @@ void Board_LTC6804_GetCellVoltages(BMS_PACK_STATUS_T* pack_status) {
...
+    // Get thermistor voltages
+    // if flag is not true, return
+    if (!ltc6804_getThermistorVoltagesFlag) {
+        return;
+    }
+    
+    uint32_t gpioVoltages[MAX_NUM_MODULES * LTC6804_GPIO_COUNT];
+    uint16_t i;
+    for (i=0; i<MAX_NUM_MODULES*LTC6804_GPIO_COUNT; i++) {
+        gpioVoltages[i] = 0;
```
Comment (jatron) [inline src/board.c]: I probably don't need to set the gpioVoltages array to all zeroes. I can fix this later.

**Review by jatron — COMMENTED** [review]
Good suggestions. I made these changes and pushed to the cell_temperatues branch. Should I submit another pull request or wait for review from Skanda and Rango?

**Pull request merged**